helios-system-tests/src/main/java/com/spotify/helios/system/ReapingTest.java

```diff
@@ -65,22 +67,21 @@ public void test() throws Exception {
     awaitHostRegistered(client, testHost(), LONG_WAIT_SECONDS, SECONDS);
     awaitHostStatus(client, testHost(), UP, LONG_WAIT_SECONDS, SECONDS);
 
-    int expectedExitCode = -1;
-    if (docker.info().executionDriver().startsWith("lxc-")) {
-      // with LXC, killing a container results in exit code 0
-      expectedExitCode = 0;
-    }
+    // With LXC, killing a container results in exit code 0.
+    // On ubuntu it's -1, and on boot2docker it's 137.
+    final List<Integer> expectedExitCodes = docker.info().executionDriver().startsWith("lxc-") ?
+                                            asList(0) : asList(-1, 137);
 
     // Wait for the agent to kill the container
     final ContainerExit exit1 = docker.waitContainer(intruder1);
-    assertThat(exit1.statusCode(), is(expectedExitCode));
+    assertThat(exit1.statusCode(), isIn(expectedExitCodes));
```

Comment:
That's interesting. I guess I'm down to merge this, but academically it would be nice to figure out *why* it's different.